COLLABORATOR_GUIDE.md

```diff
@@ -2,38 +2,41 @@
 
 ## Contents
 
-* [Issues and Pull Requests](#issues-and-pull-requests)
-  * [Welcoming First-Time Contributors](#welcoming-first-time-contributors)
-  * [Closing Issues and Pull Requests](#closing-issues-and-pull-requests)
-  * [Author ready pull requests](#author-ready-pull-requests)
-  * [Handling own pull requests](#handling-own-pull-requests)
-* [Accepting Modifications](#accepting-modifications)
-  * [Code Reviews](#code-reviews)
-  * [Consensus Seeking](#consensus-seeking)
-  * [Waiting for Approvals](#waiting-for-approvals)
-  * [Testing and CI](#testing-and-ci)
-    * [Useful CI Jobs](#useful-ci-jobs)
-    * [Starting a CI Job](#starting-a-ci-job)
-  * [Internal vs. Public API](#internal-vs-public-api)
-  * [Breaking Changes](#breaking-changes)
-    * [Breaking Changes and Deprecations](#breaking-changes-and-deprecations)
-    * [Breaking Changes to Internal Elements](#breaking-changes-to-internal-elements)
-    * [Unintended Breaking Changes](#unintended-breaking-changes)
-      * [Reverting commits](#reverting-commits)
-  * [Introducing New Modules](#introducing-new-modules)
-  * [Additions to N-API](#additions-to-n-api)
-  * [Deprecations](#deprecations)
-  * [Involving the TSC](#involving-the-tsc)
-* [Landing Pull Requests](#landing-pull-requests)
-  * [Using `git-node`](#using-git-node)
-  * [Technical HOWTO](#technical-howto)
-  * [Troubleshooting](#troubleshooting)
-  * [I Made a Mistake](#i-made-a-mistake)
-  * [Long Term Support](#long-term-support)
-    * [What is LTS?](#what-is-lts)
-    * [How are LTS Branches Managed?](#how-are-lts-branches-managed)
-    * [How can I help?](#how-can-i-help)
-* [Who to CC in the issue tracker](#who-to-cc-in-the-issue-tracker)
+- [Node.js Collaborator Guide](#nodejs-collaborator-guide)
```

Comment:
Is there a reason to change `*` to `-`? If I'm not mistaken, they are equivalent in markdown and our linter enforces `*` over `-` so I would expect this to cause an error when running `make lint-md`.

Reply:
These were apparently changed automatically by one of the vscode plugins I have installed. Very annoying. I'll undo these

Reply:
Oh, sure, blame it on the plugin. A LIKELY STORY!